cuegui/cuegui/Constants.py

```diff
@@ -32,7 +32,12 @@
 import opencue
 
 
-VERSION = "1.3.0"
+possible_version_path = os.path.join(os.path.abspath(os.path.join(__file__ ,"../../..")), 'VERSION.in')
```

Comment:
nit: Make sure that there is a single space in between arguments in function calls.

So it should be `os.path.join(__file__ , "../../..")` (adding a space after the comma).